### DISCUSSION TITLE
Github Copilot User Authorization Error.

### DISCUSSION THREAD

**Discussion by harkaran911** [community/community#11004] [Copilot Conversations] [answered]
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized".
How can I fix this problem? thanks.

**Comment by davecheney** [comment, marked as answer]
This error indicates that you are not part of the beta invite list. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060

**Reply by harkaran911** [reply]
But in how many time do they give Authorization?

**Reply by davecheney** [reply]
We're inviting new users as we have capacity.

**Reply by Aish-Luci** [reply]
I already has access to github co-pilot, but now this error is coming.

**Reply by chrismochinski** [reply]
Same issue - I've been using it for months and now this error.

**Reply by edmilson-dk** [reply]
I also just got this same error in VsCode with the copilot alerting the error "User not authorized" but I've had access to the copilot for months and just now I got this error.

**Reply by drewjbartlett** [reply]
Same here.

**Reply by BirdMachine** [reply]
Same :< had access before. Cant re-enable via billing, I'm guessing because my total's $0.

**Reply by chrismochinski** [reply]
I'm back up and running!

Disable CoPilot in VSCode
Quit VSCode
Restart VSCode
Re-enable Copilot

If you get the same error...

Follow the link in the error to the payment stuff and click the small link below the pricing options to set up your payment later. 
Follow the prompts [and link](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) to install CoPilot, even if it's already installed
Quit VSCode again
Click the green Install button on the above link. Let it open up VSCode and it should kick back on!

This worked for me. I hope it works for more folks!
